### PR TITLE
cloud: ovirt: add affinity groups module

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py

##### ANSIBLE VERSION

```
ansible 2.3.0
```

##### SUMMARY

This PR adds new module which handle managing of affinity groups in oVirt.

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina @ryansb

**Comment by machacekondra** [comment]
shipit

File: lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        default: present
+    cluster:
+        description:
+            - "Name of the cluster of the affinity group."
+    description:
+        description:
+            - "Description of the affinity group."
+    host_enforcing:
+        description:
+            - "If I(true) vm cannot start on host if it does not satisfy the C(host_rule)."
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py]: vm -> VM

File: lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+            - "C(This parameter is support since oVirt 4.1 version.)"
+    host_rule:
+        description:
+            - "If I(positive) I(all) vms in this group should run on the this host."
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py]: vms -> VMs

File: lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+            - "If I(negative) I(no) vms in this group should run on the this host."
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py]: vms -> VMs

**Review by mwperina — CHANGES_REQUESTED** [review]

**Comment by machacekondra** [comment]
@ryansb shipit

File: lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py (hunk 1 above)
Comment (mwperina) [reply lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py]: Above comment is still valid :-)

**Review by mwperina — CHANGES_REQUESTED** [review]

**Pull request merged**